### DISCUSSION TITLE
Quarto - Quarto Dashboards Demonstration

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11415] [Blog]
# Quarto - Quarto Dashboards Demonstration

A brief tour of how to build a dashboard in Quarto v1.4.

https://quarto.org/docs/blog/posts/2023-12-07-quarto-dashboards-demo/

**Comment by mad0perator** [comment]
Just now seeing this. Can't wait to try it out right now!
